bin/templates/cordova/Api.js

```diff
@@ -29,11 +29,23 @@ var AndroidProject = require('./lib/AndroidProject');
 var PlatformMunger = require('cordova-common').ConfigChanges.PlatformMunger;
 var PluginInfoProvider = require('cordova-common').PluginInfoProvider;
 
-var ConsoleLogger = require('./lib/ConsoleLogger');
 var pluginHandlers = require('./lib/pluginHandlers');
+var CordovaLogger = require('cordova-common').CordovaLogger;
+var selfEvents = require('cordova-common').events;
 
 var PLATFORM = 'android';
 
+function setupEvents(externalEventEmitter) {
```

Comment:
Could you make this function return the emitter instance? `setupEvents(events); events = events || selfEvents;` -> `events = setupEvents(events);`